modules/packages/BLAS.chpl

```diff
@@ -34,7 +34,7 @@ describes BLAS as the following::
 
 This module wraps the functionality of level 3 matrix-matrix BLAS routines,
```

Comment:
Going to change it to simply "BLAS routines", since BLAS-2 is right around the corner.